metricshub-internaldb-extension/src/main/java/org/sentrysoftware/metricshub/extension/internal/db/InternalDbExtension.java

```diff
@@ -34,29 +34,32 @@
 
 /**
  * This class implements the {@link ISourceComputationExtension} contract, reports the supported features,
- * processes Sql sources.
+ * processes LocalSql sources.
```

Comment:
Yeah this should be more like `InternalDbQuery` rather than `LocalSql`, which is really confusing if the user is monitoring a MySQL instance running on localhost.

So it's either `InternalDbQuery`, or `InternalSqlQuery`, or `InternalDbSql`, or `InternalSql`.

Personally, I prefer `InternalDbQuery`, but I'll let you choose from the above options! @NassimBtk 😉

Reply:
Yeah InternalDbQuery is good 😉.